en/developer_guide/logs.rst

```diff
@@ -0,0 +1,90 @@
+Logs
+====
+
+Our backoffice logging is based on Monolog.
+We provide an interface to display all the user action on the website
+It displays the date and time, user's IP, user's name, the current site's name and the log's message.
+
+Configuration
+-------------
+
+Monolog uses channels to record the log. To link a channel and a record,you will have to define an handler.
+The configuration of the handler can be placed in the app/config/log.yml file.
+
+To save the logs in MongoDB, We need to define a handler
```

Comment:
Never use ``We`` in a documentation, I will recommand you to prefer the use of ``you``